lib/src/widgets/chat/typing/ds_typing_message_bubble.widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+/// Use [align] left or right to position typing on screen
```

Comment:
Add a single-sentence summary of what this widget actually does, before going into details of implementation like the usage of align property.

Reference: https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary